tinyfilemanager.php

```diff
@@ -3954,7 +3954,7 @@ function fm_show_header()
       <div class="modal-dialog" role="document">
         <form class="modal-content rounded-3 shadow <?php echo fm_get_theme(); ?>" method="post" autocomplete="off">
           <div class="modal-body p-4 text-center">
-            <h5 class="mb-3"><?php echo lng('Are you sure want to rename?') ?></h5>
+            <h5 class="mb-3"><?php echo lng('Are you sure you want to rename?') ?></h5>
```

Comment:
Adding the subject ('you') is an incomplete fix. It needs also the clause 'that'. In this example, which applies to all sentences in this PR:
`Are you sure you want(...)`
Should instead be
`Are you sure that you want(...)`

Reply:
> Adding the subject ('you') is an incomplete fix. It needs also the clause 'that'. In this example, which applies to all sentences in this PR: `Are you sure you want(...)` Should instead be `Are you sure that you want(...)`

@ner00 To be fair, "that" is often omitted when the clause functions as the direct object. It's a matter of style, not grammar. Omission is pretty common in this context too, for instance Mac OS' _Finder_ prompts similarly: "Are you sure you want to delete ...".